packages/order/src/utils/transform-order.ts

```diff
@@ -29,6 +29,19 @@ export function formatOrder(
       }
     })
 
+    order.shipping_methods = order.shipping_methods?.map((shippingMethod) => {
+      const sm = { ...shippingMethod.shipping_method }
```

Comment:
question: There is likely a good answer to this question, but I might have missed it: why does the shipping method have a `shipping_method`?

Reply:
Nevermind, I realise it was introduced in this pr 😄

Reply:
"shipping_methods" is now a relation table linking order_id + shipping_method + version.

this util is just to reshape the final response of the module, where you receive the shipping_method entity, not the "link" table.